### PR TITLE
use circleci for chart testing

### PR DESCRIPTION
**Pull request by rimusz**

### CODE REVIEW

**Comment by rimusz** [comment]
@arschles doesn't look that CircleCI is enabled

**Comment by marwan-at-work** [comment]
@rimusz Thanks for the PR :) 

Out of curiosity why CircleCI and not DroneCI? Can it not test Helm charts?

**Comment by rimusz** [comment]
I love DroneCI, but it doesn't support `privileged` docker containers which are need for Helm chart testing install in [KinD](https://kind.sigs.k8s.io) which is Kubernetes in docker.

**Comment by marwan-at-work** [comment]
@rimusz got it. We can enable CircleCI on this repo if this looks good to everyone:
cc: @arschles

**Comment by rimusz** [comment]
@arschles is fine of enabling it https://github.com/gomods/athens/issues/1216

**Comment by marwan-at-work** [comment]
@rimusz I re-enabled CircleCI webhooks, feel free to update this PR to test your change.

**Comment by rimusz** [comment]
@marwan-at-work I still don't see CircleCI

**Comment by arschles** [comment]
@rimusz I pushed a basic circleci file (https://github.com/gomods/athens/pull/1226) and ensured that builds are running. Builds for forked PRs are also on in circleci, so when you resolve the conflict the circle build should run.

The conflict should be easy to resolve - replace everything in my config file with your stuff 😁

**Comment by rimusz** [comment]
all good now 👍

File: .drone.yml [hunk 1]
```diff
@@ -4,17 +4,6 @@ name: default
...
-  image: quay.io/helmpack/chart-testing:v2.3.3
-  commands:
-  - ct lint --config test/ct.yaml
-  when:
-    branch:
-      - master
-    event:
-      - push
-      - pull_request
-
```
Comment (arschles) [inline .drone.yml]: @rimusz is the `ct lint --config test/ct.yaml` command replaced somewhere in the e2e-kind script? I don't see it exactly but it looks close, so just making sure

Comment (rimusz) [reply .drone.yml]: this does run the chart lint part which doesn't need kind.
`e2e-kind.sh` has `ct install --config test/ct.yaml`
Also we can do just in `kind` `ct lint-and-install --config test/ct.yaml` which will do both.
Let me know if you this change.

Comment (rimusz) [reply .drone.yml]: https://github.com/gomods/athens/pull/1225/commits/12901a8d0550c1a3bb43ed09a589429d81757769#diff-37eda1430799e62f92596aa54fae0596R85

**Comment by arschles** [comment]
Thanks for this @rimusz ! Merging now

**Pull request merged**